hi-ml/src/health_ml/runner.py

```diff
@@ -156,18 +156,15 @@ def validate(self) -> None:
         """
         Runs sanity checks on the whole experiment.
         """
-        if not self.experiment_config.azureml:
+        if not self.experiment_config.cluster:
             if self.lightning_container.hyperdrive:
-                logging.info("You have turned on HyperDrive for parameter tuning. This can "
-                             "only be run in AzureML. We switched on submitting to AzureML.")
-                self.experiment_config.azureml = True
+                raise ValueError("HyperDrive for hyperparameters tuning is only supported when submitting the job to "
+                                 "AzureML. You need to specify a computer cluster to submit your job to with the "
```

Comment:
typo here ('to with')

Reply:
(Also in the cross validation check below)

Reply:
I enabled auto-merge (sorry), I will open a PR to account for your latest comments.

Reply:
No worries. Thanks!